### PR TITLE
chore: update docs for oracle v2.0.0 release

### PR DESCRIPTION
**Pull request by Yashk767**

### CODE REVIEW

File: docs/commands/staker/set-config.md [hunk 1]
```diff
@@ -37,15 +37,15 @@ Example:
 **Note**: _Make sure you input your alternate provider value as a value to alternateProvider flag
 
 ```
-$ ./razor setConfig --provider https://mainnet.skalenodes.com/v1/turbulent-unique-scheat --alternateProvider [ALTERNATE_PROVIDER] --gasmultiplier 1 --buffer 20 --wait 30 --gasprice 0 --logLevel debug --gasLimit 2 --gasLimitOverride 50000000 --rpcTimeout 10 --httpTimeout 10 --logFileMaxSize 200 --logFileMaxBackups 52 --logFileMaxAge 365
+$ ./razor setConfig --provider https://elated-tan-skat.explorer.mainnet.skalenodes.com --gasmultiplier 1 --buffer 5 --wait 1 --gasprice 0 --logLevel debug --gasLimit 2 --gasLimitOverride 30000000 --rpcTimeout 5 --httpTimeout 5 --logFileMaxSize 200 --logFileMaxBackups 10 --logFileMaxAge 60
```
Comment (yohanelly95) [inline docs/commands/staker/set-config.md]: rpc: https://mainnet.skalenodes.com/v1/elated-tan-skat

File: docs/stake.md [hunk 2]
```diff
@@ -225,6 +240,7 @@ For more details around all the commands of `razor-go`, please check out the `or
 If you would rather install from source, please follow Instructions here to [run a Razor Network node from source](https://github.com/razor-network/oracle-node#building-the-source).
 
 ## Update docker image
+> **_NOTE:_**: _If you are updating to `v2` in the same machine where `v1` node was running, make sure you delete all the files in ` .razor/data_files` directory.
```
Comment (yohanelly95) [inline docs/stake.md]: If you are updating to v2 on the same machine where the v1 node was running, make sure to delete all the files in the .razor/data_files directory.

Comment (Yashk767) [reply docs/stake.md]: Done in https://github.com/rajkharvar/docs-v2/pull/2/commits/7e9762ff5494e536401b6ea1517f441014fdfd30

File: docs/commands/staker/set-config.md (hunk 1 above)
Comment (Yashk767) [reply docs/commands/staker/set-config.md]: Fixed in https://github.com/rajkharvar/docs-v2/pull/2/commits/7e9762ff5494e536401b6ea1517f441014fdfd30

**Review by rajkharvar — APPROVED** [review]

**Review by yohanelly95 — APPROVED** [review]

**Pull request merged**